apps/backend/api/src/app/games/games.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { CreateGameDto } from './dto/create-game.dto';
+import { UpdateGameDto } from './dto/update-game.dto';
+import { Game } from '../../entities';
+
+@Injectable()
+export class GamesService {
+  constructor(
+    @InjectRepository(Game) private gamesRepository: Repository<Game>,
+  ) {}
+
+  async create(createGameDto: CreateGameDto) {
+    return await this.gamesRepository.save(createGameDto);
```

Comment:
Si je ne dis pas de bétises il ne faut pas qu'on mette de return await
```suggestion
    return this.gamesRepository.save(createGameDto);
```

Reply:
Fait